modules/installation/nav-installation-guide.adoc

```diff
@@ -80,8 +82,10 @@ include::modules/installation/pages/install-vm.adoc[leveloffset=+2]
 // Install the Server
 include::modules/installation/pages/install-server.adoc[leveloffset=+2]
 
+ifeval::[{suma-content} == true]
 // Install Proxy
-//include::modules/installation/pages/install-proxy.adoc[leveloffset=+2]
+include::modules/installation/pages/install-proxy.adoc[leveloffset=+2]
```

Comment:
It seems "install proxy unified" is missing here.

Reply:
Proxy install with the Unified Installer comes earlier.  ATM, we are sorting by the installation mode, not by product...

1. Install with Unified Installer
1.1 SUMA Server
1.2 SUMA Proxy
...
3. Install on Top of SLES
3.1 Server
3.2 Proxy

And then we have IBM z, and finally Setup. And public cloud somehow mixed in.
(Nice, but I do not like this sequence too much. I'd rather prefer strictly procedural installation and setup instructions.)